### DISCUSSION TITLE
Welcome to MARSHMELLOW-USERBOT Discussions!

### DISCUSSION THREAD

**Discussion by Marshmellow098** [Marshmellow098/MARSHMELLOW-USERBOT#3] [General]
![PicsArt_12-07-03 35 32](https://user-images.githubusercontent.com/74300487/101605142-d8c72f00-3a27-11eb-8ac3-5f800e956d9e.jpg)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by annihilatorrrr** [comment]
kya hal chal hai yar ?? 
permanent pic yahi hoga na pacca ??
yar pic change na karo merge mai issue ata hai

**Reply by Marshmellow098** [reply]
OK ya pic nahi change ho gaya only of resources and groups etc will change 😎😎.

**Reply by annihilatorrrr** [reply]
Thanks dude 😌😌 now I can use it !

**Reply by Marshmellow098** [reply]
_### Do today morning gm everyone from my side also.😪😪😪_
